docs/guide/data_types/timeseries.rst

```diff
@@ -192,7 +192,7 @@ the `~sunpy.timeseries.timeseriesbase.GenericTimeSeries.data` attribute.  The
 data is implemented as a Pandas `~pandas.DataFrame`, so to get a look at what
 data you have available use: ::
 
-    >>> my_timeseries.data  # doctest: +SKIP
+    >>> my_timeseries.data
```

Comment:
isn't this going to break because it's expecting output? Surely we want the skip here?

Reply:
Odd how this all seemed to pass locally but it shouldnt.